apps/vocabulary-operations/intersection.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	metrics "github.com/googleapis/gnostic/metrics"
+)
+
+func mapIntersection(v *metrics.Vocabulary) {
```

Comment:
This method looks identical to one in main.go - should we try to consolidate these?

Reply:
I first created intersection in the vocabulary-difference app. Tim and I had the idea to build upon it and create other operators which are now all in the vocabulary-operations app. I believe in my next pull I'll be deleted vocabulary-difference so there shouldn't be any more duplicates